src/main/groovy/io/github/groovymc/modsdotgroovy/ModsDotGroovy.groovy

```diff
@@ -56,25 +56,39 @@ class ModsDotGroovy implements Plugin<Project> {
             configuration.dependencies.add(project.dependencies.create(ext.mdgDsl()))
 
             if (ext.automaticConfiguration.get()) {
-                final srcSets = project.extensions.getByType(JavaPluginExtension).sourceSets
```

Comment:
What happened to the brackets?

Reply:
I have no clue. Looks like I shifted some stuff around and IntelliJ butchered it